docs/examples/harb.mdx

```diff
@@ -142,84 +99,128 @@ Defines the spawn areas.
 </spawns>
 ```
 
+[<i className="icon-right">➡️</i>](/docs/modules/mechanics/filters)
+Define the filters on the map and determine which events are allowed.
+
+```xml
+<filters>
+    <!-- Always deny TNT placement, destruction, or interaction -->
```

Comment:
i feel this is misleading, because it makes you2 think that the filter is what prevents the place/interact, when really what does that, is the <apply block="deny-tnt"> etc
maybe re-write this comment to better reflect that?